common/types/post.go

```diff
@@ -6,6 +6,7 @@ type PostInfo struct {
 	CommitmentATX ATXID
 	Nonce         *VRFPostIndex
 
+	Challenge     []byte
```

Comment:
`Challenge` is part of the proof, not POST metadata, but this structure is returned from `Info()` which only contains the post metadata.